### ISSUE TITLE
Accessibility - White DOL Seal on Gold Background Does Not Have Enough Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#199]
While looking at something else, I noticed that the white DOL seal in the header section has poor color contrast with the gold colored background.

Not sure why this was changed along with other items being now gold and white.

Please see the attached screenshot.

<img width="960" alt="comply chain - topic pages - topic text and check box do not have enough color contrast" src="https://user-images.githubusercontent.com/10423487/125525724-86035eed-a31b-44f1-9f18-8b8ab6ed557e.png">

**Comment by jfc3-dol** [comment]
@pbhatt17 
I tested the a random page on both Android and the web versions, and the color contrast is back for the DOL seal to what it used to be and again passes.

So I'm going to close this issue.
